sklearn/linear_model/tests/test_ridge.py

```diff
@@ -1596,6 +1596,15 @@ def test_ridge_sag_with_X_fortran():
     Ridge(solver="sag").fit(X, y)
 
 
+def test_ridge_alpha_boundary_warning():
+    ridge = RidgeCV(alphas=[0.1, 1])
+    X, y = X_diabetes, y_diabetes
+    with pytest.warns(
+        ConvergenceWarning, match="lies at a boundary of the explored range"
+    ):
+        ridge.fit(X, y)
```

Comment:
To follow-up with @thomasjpfan's comment: https://github.com/scikit-learn/scikit-learn/pull/21303/files#r814000544

Would this completion do?
```suggestion
def test_ridge_alpha_boundary_warning():
    ridge = RidgeCV(alphas=[0.1, 1])
    X, y = X_diabetes, y_diabetes
    with pytest.warns(
        ConvergenceWarning, match="Some of the individual targets"
    ):
        ridge.fit(X, y)

    ridge = RidgeCV(alphas=1, alpha_per_target=True)
    X, y = X_diabetes, y_diabetes
    with pytest.warns(
        ConvergenceWarning,
        match="The optimal value for the regularization parameter"
    ):
        ridge.fit(X, y)        
```